modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -878,6 +879,10 @@ performance, this value should not be greater than the number of available proce
 |Whether or not strings should be trimmed for whitespaces.
 |false
 
+|--update-all-matching-relationships
+|label:new[Introduced in 2025.01] If one relationship data entry matches multiple existing relationships, this decides whether to update all matching, or to instead log as error.
```

Comment:
Something is odd with this message. When this option is enabled, the import updates all relationships that match the relationship data entry. Otherwise, it logs an error (when false, I guess). Is that what is meant by decides?

Reply:
Yes that's correct

Reply:
Can we amend the description in the code so it is clearer what it means?

Reply:
@renetapopova, I know that the description needs to be amended. That is on my list. I'm going to fix that when we finally merge this PR.